corehq/apps/export/templates/export/incremental_export.html

```diff
@@ -107,5 +112,38 @@ <h3>
         </div>
       </div>
     </td>
+    <td>
+      <button type="button"
+              data-toggle="modal"
+              data-bind="attr: {'data-target': '#reset-incremental-export-' + id}"
+              class="btn btn-warning">
+        <i class="fa fa-recycle"></i> {% trans "Resend all cases" %}
+      </button>
+
+      <div class="modal fade"
+           data-bind="attr: {id: 'reset-incremental-export-' + id}">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+              <h3>
+                {% trans "Resend all cases for " %} <strong data-bind="text: name"></strong>?
+              </h3>
+            </div>
+            <div class="modal-footer">
+              <button type="button"
+                      class="btn btn-default"
+                      data-dismiss="modal">
+                {% trans "Cancel" %}
+              </button>
+              <a data-bind="attr: {href: $parents[1].reverse('incremental_export_resend_all', id)}"
```

Comment:
What does `$parents[1]` mean? Would `$root` be equivalent here?

Reply:
yes, thanks!